jekyll/_cci2/configuration-reference.md

```diff
@@ -716,6 +717,8 @@ jobs:
 
 <sup>(3)</sup> _This resource is available only for customers with an annual contract. [Open a support ticket](https://support.circleci.com/hc/en-us/requests/new) if you would like to learn more about our annual plans._
 
+<sup>(4)</sup> _This resource requires a minimum 24-hour lease. See the [Dedicated Host for macOS]({{ site.baseurl }}/2.0/dedicated-hosts-macos) document to learn more about this resource class._
```

Comment:
```suggestion
<sup>(4)</sup> _This resource requires a minimum 24-hour lease. See the [Dedicated Host for macOS]({{ site.baseurl }}/2.0/dedicated-hosts-macos) document to learn more about this resource class._
```

Reply:
Maybe we want to say "page" instead of "document"?

Reply:
Good point. I've updated it.